pkgs/tools/system/zcfan/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+# Testing this requires a Thinkpad or the presense of /proc/acpi/ibm/fan
+stdenv.mkDerivation rec {
+  pname = "zcfan";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "cdown";
+    repo = "zcfan";
+    rev = version;
+    hash = "sha256-XngchR06HP2iExKJVe+XKBDgsv98AEYWOkl1a/Hktgs=";
+  };
+
+  doInstallCheck = true;
+
+  configurePhase = ''
```

Comment:
nitpick: shouldn't this go in `postPatch` ?

Reply:
Does fit better there

https://github.com/NixOS/nixpkgs/pull/253579/commits/1d558d148854d28354aa5b8279b43a2442f1aac9